README.md

```diff
@@ -2,4 +2,43 @@
 
 [![Go](https://github.com/evenlab/go-kit/actions/workflows/go.yml/badge.svg)](https://github.com/evenlab/go-kit/actions/workflows/go.yml)
 
-There are helpful lightweight and tested functionality for reuse in other projects
+### Golang KIT of helpful lightweight and tested functionality for reuse in other projects
+
+# `Makefile` Usage on Windows
```

Comment:
Semantically, headers (starts with '#') shouldn't contain sections with code (starts with ` or ```)
I suggest removing them and rephrasing with: Usage GNU Make on Windows;  Setup Choco package manager;

Reply:
I'm not sure that ` is for blocks of code, but to highlight some text I guess...